packages/database/src/migrations/20201015015257-DeleteAnswerWith9998-modifies-data.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return db.runSql(`
+    DELETE FROM "answer" WHERE text = '9998';
```

Comment:
My only concern here is that this could be too broad. 

I would feel a lot more comfortable if we also asserted that the survey was `Reproductive Health Combined`.
Something like: 
```suggestion
    DELETE FROM "answer" a
    using survey_response sr
    join survey s on s.id = sr.survey_id 
    where a.survey_response_id = sr.id 
    and a.text = '9998'
    and s.name = 'Reproductive Health Combined';
```

Reply:
Thanks Alastair, that's a good point. I haven't thought about it! This is also a nice query writing.